Dockerfile.7.x-cli

```diff
@@ -9,6 +9,7 @@ ARG PHP_VERSION
 LABEL php.version=${PHP_VERSION}
 RUN set -eux \
     &&  export http_proxy=${HTTP_PROXY} \
+    &&  if [[ $PHP_MINOR_VERSION -gt 1 ]]; then echo "PHP_VERSION > 7.1" && printf "Package: *\nPin: origin packages.sury.org\nPin-Priority: 100\n" > /etc/apt/preferences.d/php ; fi \
```

Comment:
Condition/test not necessary : this recipe uses Sury's repo thus always need the APT pinning (for every version provided by Sury).

Reply:
It's a debug for the pinning bug.